### ISSUE TITLE
Keeping texture uv coordinates 

### ISSUE THREAD

**Issue by talkenig** [mkazhdan/PoissonRecon#187]
Hi,
I have texture uv coordinates with my input points (two floating point numbers in the range 0-1 per vertex). Is there a way to keep them as part of the recon in a similar manner to the way the code treats input RGB per vertex? The main problem, of course, is that the RGB are expected to be UINT.
Thanks

**Comment by mkazhdan** [comment]
Yes. That should be doable if your input us in .ply format (so the file itself describes the storage type). However, you should note that the resulting values will be blends of the input values, which may give undesirable results if the texture values are discontinuous. (e.g. if you are using a multi-chart atlas).

**Comment by talkenig** [comment]
Thanks! So should the uv values be named as R G and set B as an arbitrary
value just specified as float, or do I need to name those U V, or perhaps
it doesn't matter?

**Comment by mkazhdan** [comment]
You should be able to call them whatever you want, since the .ply file encodes the description.
For example, your ply file header could looks something like:

ply
format binary_little_endian 1.0
element vertex <vertex num>
property float x
property float y
property float z
property float nx
property float ny
property float nz
property float u
property float v
end_header

where <vertex num> is the number of point samples.
